sesman/chansrv/chansrv.c

```diff
@@ -1792,32 +1765,54 @@ main(int argc, char **argv)
     enum logReturns error;
     struct log_config logconfig;
     enum logLevels log_level;
-    char *restrict_outbound_clipboard_env;
     g_init("xrdp-chansrv"); /* os_calls */
 
+    g_snprintf(default_sesman_ini, 255, "%s/sesman.ini", XRDP_CFG_PATH);
```

Comment:
I found this.

Reply:
That's the default value if nothing is specified. I'll add a comment and I can also remove the hard-coded limit.